src/seismometer/_api.py

```diff
@@ -1447,7 +1447,7 @@ class ExploreCohortEvaluation(ExplorationCohortSubclassEvaluationWidget):
 
     Creates a 2x3 grid of individual performance metrics across cohorts.
 
-    Plots include Sensitivity, Flagged, PPV, Specificity, NPV vs Thresholds.
+    Plots include Sensitivity, Flag Rate, PPV, Specificity, NPV vs Thresholds.
```

Comment:
Can you clean the metadata from the example notebook, spotted because it references 'Flagged'
From some saved snippets, I think it's basically running this and commiting:
```shell
jupyter nbconvert \
    --ClearMetadataPreprocessor.enabled=True \
    --ClearMetadataPreprocessor.clear_cell_metadata=False \
    --clear-output \
    --to=notebook --inplace  **FILE**
```

Reply:
FINC